### ISSUE TITLE
npm start error

### ISSUE THREAD

**Issue by pierluigizagaria** [codetheweb/muse#475]
```
PS D:\Files\Muse> npm run start

> muse@0.3.1 start
> npm run env:set-database-url -- node --experimental-json-modules dist/src/scripts/migrate-and-start.js


> muse@0.3.1 env:set-database-url
> node dist/src/scripts/run-with-database-url.js "node" "--experimental-json-modules" "dist/src/scripts/migrate-and-start.js"

node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///D:/Files/Muse/dist/package.json" needs an import assertion of type "json"
    at new NodeError (node:internal/errors:371:5)
    at validateAssertions (node:internal/modules/esm/assert:82:15)
    at defaultLoad (node:internal/modules/esm/load:25:3)
    at ESMLoader.load (node:internal/modules/esm/loader:359:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:280:58)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:297:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:261:34)
    at async ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:81:21)
    at async Promise.all (index 1) {
  code: 'ERR_IMPORT_ASSERTION_TYPE_MISSING'
}

Node.js v17.3.1
file:///D:/Files/Muse/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js
    at makeError (file:///D:/Files/Muse/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///D:/Files/Muse/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///D:/Files/Muse/dist/src/scripts/run-with-database-url.js:6:5 {
  shortMessage: 'Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  command: 'node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  escapedCommand: 'node --experimental-json-modules "dist/src/scripts/migrate-and-start.js"',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}

Node.js v17.3.1
```
I'm having this error when starting the bot on windows

**Comment by pierluigizagaria** [comment]
Having the same issue

**Comment by pierluigizagaria** [comment]
Was using nodejs 17, downgraded to 16 fix that error
```

```

**Comment by pierluigizagaria** [comment]
Backslashes on the prisma path are breaking the windows migration. 
Found a workaround by putting the sqlite path with forward slashes `/`

```
datasource db {
  provider = "sqlite"
  url      = "file:D:/Files/Muse/data/db.sqlite"
}
```

**Comment by codetheweb** [comment]
I believe this should be fixed in v0.3.2, please try it out and let me know.

**Comment by lemns** [comment]
This is till an ongoing issue

```
PS D:\MUSIC\muse-0.5.2> npm run start

> muse@0.5.2 start
> npm run env:set-database-url -- node --experimental-json-modules dist/src/scripts/migrate-and-start.js


> muse@0.5.2 env:set-database-url
> node dist/src/scripts/run-with-database-url.js "node" "--experimental-json-modules" "dist/src/scripts/migrate-and-start.js"

(node:13056) ExperimentalWarning: Importing JSON modules is an experimental feature. This feature could change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
========================================
========================================
================= muse =================
========================================
=========== Made with 🎶 by ============
==== https://github.com/codetheweb =====
========================================
========================================

Running version 0.5.2 built on unknown date (commit unknown)

🔧 Need help? https://github.com/codetheweb/muse/discussions/new
🐛 Bug? https://github.com/codetheweb/muse/issues/new
✨ New feature? https://github.com/codetheweb/muse/issues/new
☕ Discussion? https://github.com/codetheweb/muse/discussions/new
💰 Help me? https://www.paypal.me/codetheweb https://github.com/sponsors/codetheweb


✖ Failed to apply database migrations:
Error: P1012: error: Error validating: This line is not a valid definition within a datasource.  
  -->  schema.prisma:4
   |
 3 |                 provider = "sqlite"
 4 |                 url = "file:D:\MUSIC\muse-0.5.2\data\db.sqlite"
 5 |             }
   |

file:///D:/MUSIC/muse-0.5.2/node_modules/execa/lib/error.js:59
                error = new Error(message);
                        ^

Error: Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js
    at makeError (file:///D:/MUSIC/muse-0.5.2/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///D:/MUSIC/muse-0.5.2/node_modules/execa/index.js:119:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async file:///D:/MUSIC/muse-0.5.2/dist/src/scripts/run-with-database-url.js:6:5 {
  shortMessage: 'Command failed with exit code 1: node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  command: 'node --experimental-json-modules dist/src/scripts/migrate-and-start.js',
  escapedCommand: 'node --experimental-json-modules "dist/src/scripts/migrate-and-start.js"',    
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
PS D:\MUSIC\muse-0.5.2>

```
